src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String COMMAND_TODO = "TODO";
+    private static final String COMMAND_DEADLINE = "DEADLINE";
+    private static final String COMMAND_EVENT = "EVENT";
+    private static final String COMMAND_LIST = "LIST";
+    private static final String COMMAND_BYE = "BYE";
+    private static final String COMMAND_DONE = "DONE";
+
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+
+    public static void printHello() {
+        printLine();
+        System.out.println("Greetings Peasants! I'm the Almighty Allfather!");
+        System.out.println("What can I do for you today?");
+        printLine();
+    }
+
+    public static void printBye() {
+        printLine();
+        System.out.println("Begone!");
+        printLine();
+    }
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(String commandType, String commandArgs) {
+        Task t = null;
```

Comment:
Should Task t have a better variable name?

Reply:
Thank you, I shall implement a better name :)